src/lightning_app/core/api.py

```diff
@@ -70,21 +69,18 @@ class SessionMiddleware:
 
 
 class UIRefresher(Thread):
-    def __init__(self, api_publish_state_queue, api_response_queue, refresh_interval: float = 0.1) -> None:
```

Comment:
This is not a timeout.

The PR does not explain why we remove this.

Reply:
Rephrase to: `Remove the sleep call within the APIRefresher`